conans/test/functional/toolchains/meson/test_meson.py

```diff
@@ -296,3 +296,60 @@ def build(self):
     client.run("build .")
     assert "unrecognized character escape sequence" not in str(client.out)  # if Visual
     assert "unknown escape sequence" not in str(client.out)  # if mingw
+
+
+@pytest.mark.tool("meson")
```

Comment:
This doesn't need to be a full functional test that needs meson. The ``MesonToolchain`` can be safely operate without Meson installed at all. I think downgrading this to an integration test, or in this case, even better, a unittest, would be better.

Reply:
You're right, but I thought that it was safer if Meson minimally reads the native configuration file just to be sure that we're not making some syntax mistakes. This test is not compiling anything; it's only calling Meson to read that file. Anyway, I'm not opposed to converting it to a unit test.

Reply:
The test with Meson reading different options values is already there, in ``test_meson.py``:
```python
tc.project_options["STRING_DEFINITION"] = "Text"
tc.project_options["TRUE_DEFINITION"] = True
tc.project_options["FALSE_DEFINITION"] = False
tc.project_options["INT_DEFINITION"] = 42
tc.project_options["ARRAY_DEFINITION"] = ["Text1", "Text2"]
```

This test was about launching an error message if options is being passed, because it was erroring anyway, so it is enough to cover the error case, the other was already covered.

Reply:
You're right. I did not look hard enough to see that test was already there. Thanks!